### PR TITLE
Staging 0.39.1

### PR DESCRIPTION
**Pull request by Pavan-Gunda**
> [!warning]
> **This is a public repository, ensure not to disclose:**
>
> - [ ] personal data beyond what is necessary for interacting with this pull request, nor
> - [ ] business confidential information, such as customer names.

### What kind of PR is this?

**Required**: Mark one of the following that is applicable:

- [ ] kind/feature       
- [x] kind/improvement   
- [ ] kind/deprecation   
- [ ] kind/documentation 
- [ ] kind/clean-up      
- [ ] kind/bug           
- [ ] kind/other         

*Optional*: Mark one or more of the following that are applicable:

> [!important]
> Breaking changes should be marked `kind/admin-change` or `kind/dev-change` depending on type
> Critical security fixes should be marked with `kind/security`

- [ ] kind/admin-change 
- [ ] kind/dev-change   
- [ ] kind/security     
- [ ] [kind/adr]()      

### What does this PR do / why do we need this PR?

...

- Fixes #

#### Checklist

- [x] Proper commit message prefix on all commits
  
- Change checks:
  - [ ] The change is transparent
  - [ ] The change is disruptive
  - [ ] The change requires no migration steps
  - [ ] The change requires migration steps
  - [ ] The change upgrades CRDs
  - [ ] The change updates the config *and* the schema
- Metrics checks:
  - [ ] The metrics are still exposed and present in Grafana after the change
  - [ ] The metrics names didn't change (Grafana dashboards and Prometheus alerts are not affected)
  - [ ] The metrics names did change (Grafana dashboards and Prometheus alerts were fixed)
- Logs checks:
  - [ ] The logs do not show any errors after the change
- Pod Security Policy checks:
  - [ ] Any changed pod is covered by Pod Security Admission
  - [ ] Any changed pod is covered by Gatekeeper Pod Security Policies
  - [ ] The change does not cause any pods to be blocked by Pod Security Admission or Policies
- Network Policy checks:
  - [ ] Any changed pod is covered by Network Policies
  - [ ] The change does not cause any dropped packets in the `NetworkPolicy Dashboard`
- Audit checks:
  - [ ] The change does not cause any unnecessary Kubernetes audit events
  - [ ] The change requires changes to Kubernetes audit policy
- Falco checks:
  - [ ] The change does not cause any alerts to be generated by Falco
- Bug checks:
  - [ ] The bug fix is covered by regression tests

### CODE REVIEW

File: config/schemas/config.yaml [hunk 1]
```diff
@@ -5344,6 +5353,26 @@ properties:
...
+      fluxv2:
+        properties:
+          enabled:
+            default: false
+            type: boolean
+      kafka:
+        properties:
+          enabled:
+            default: false
+            type: boolean
```
Comment (aarnq) [inline config/schemas/config.yaml:5375]:
```suggestion
```
This should not be configured under `certmanager`.

**Review by aarnq — CHANGES_REQUESTED** [review]

File: config/schemas/config.yaml (hunk 1 above)
Comment (Pavan-Gunda) [reply config/schemas/config.yaml]: I removed them.

**Review by aarnq — APPROVED** [review]

File: config/config/sc-config.yaml [hunk 2]
```diff
@@ -60,6 +60,7 @@ objectStorage:
 
   ## Off-site backup replication between two providers or regions using rclone sync
   sync:
+    activeDeadlineSeconds: 7200
```
Comment (viktor-f) [inline config/config/sc-config.yaml:63]: This comments says that you should increase this further. https://github.com/elastisys/ck8s-issue-tracker/issues/297#issuecomment-2196716260

Comment (Pavan-Gunda) [reply config/config/sc-config.yaml:63]: Oh.. I missed that. I'll increase it.

Comment (Pavan-Gunda) [reply config/config/sc-config.yaml]: Done

**Review by viktor-f — APPROVED** [review]

**Pull request merged**